README.md

```diff
@@ -1,42 +1,41 @@
 # Acceptance tests for Jenkins
 
-A project to flesh out some of the [manual test cases for Jenkins
-LTS](https://wiki.jenkins-ci.org/display/JENKINS/LTS+RC+Testing) in an automated fashion.
+End to end test suite for Jenkins automation server and its plugins.
 
-Following areas are still worked on:
-
- * Vagrant-based `JenkinsController` implementations
-
-master | stable
------- | ------
-[![Build Status](https://jenkins.ci.cloudbees.com/buildStatus/icon?job=core/acceptance-test-harness)](https://jenkins.ci.cloudbees.com/job/core/job/acceptance-test-harness/) | [![Build Status](https://jenkins.ci.cloudbees.com/buildStatus/icon?job=core/acceptance-test-harness-stable)](https://jenkins.ci.cloudbees.com/job/core/job/acceptance-test-harness-stable/)
+The scenarios are described as unit tests controlling Jenkins through UI / REST API. Clean instance is started for
+individual tests to isolate the tests. The harness provides convenient docker support so integration tests can be written
+easily.
 
 ## Getting Started
 
-Let's start by running tests locally with `JENKINS_WAR=path/to/your/jenkins.war mvn test`.
-This runs the entire test suite with the specified Jenkins war and Firefox, so it will take a while.
-
-Set `BROWSER=chrome` and install http://code.google.com/p/chromedriver/downloads/list in `$PATH` if desired. (But tests may fail.)
-
-There is a bit of a delay since we bring up Jenkins for every single test, with
-it's own sandboxed workspace.
+The simplest way to start the harness is calling `BROWSER=firefox JENKINS_VERSION=latest mvn test`. Complete test suite
+takes hours to run due to the number of covered components/use-cases, the cost of Jenkins setup and selenium interactions.
 
 ## Further Reading
 
 ### Running tests
 
-* [Selecting browser](docs/BROWSER.md)
-* [Selecting how to launch Jenkins under test (JUT)](docs/CONTROLLER.md)
-* [Running one test](docs/SINGLE-TEST.md)
-* [Using a http proxy](docs/USING-A-HTTP-PROXY.md)
-* [Prelaunch JUT](docs/PRELAUNCH.md)
+The harness provides a variety of ways to configure the execution including:
+
+* [Selecting web browser](docs/BROWSER.md)
+* [Specifing test(s) to run](docs/SINGLE-TEST.md)
 * [Managing the versions of Jenkins and plugins](docs/SUT-VERSIONS.md)
+* [Using a http proxy](docs/USING-A-HTTP-PROXY.md)
+* [Prelaunchint JUT](docs/PRELAUNCH.md)
+* [Selecting how to launch Jenkins under test (JUT)](docs/CONTROLLER.md)
 * [Obtaining a report of plugins that were exercised](docs/EXERCISEDPLUGINSREPORTER.md)
-* [Investigation](docs/INVESTIGATION.md)
 * [Running tests in container](docs/DOCKER.md)
 * Selecting tests based on plugins they cover (TODO)
 
 ### Writing tests
+
+Given how log does it take for the suite to run, test authors are advised to focus on the most popular plugins and
+use-cases to maximize the value of the test suite. Tests that can or already are written as a part of core/plugin tests
```

Comment:
Probably a good place to discuss what is and is not practical to do with `JenkinsRule` and thus what the reasons would be for writing something using ATH.